<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
     <property name="geometry">
         <rect>
             <x>0</x>
             <y>0</y>
             <width>525</width>
             <height>399</height>
         </rect>
     </property>
     <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
         </sizepolicy>
     </property>
     <property name="windowTitle">
         <string>Form</string>
     </property>
     <widget class="QLineEdit" name="volume_edit">
         <property name="geometry">
             <rect>
                 <x>250</x>
                 <y>60</y>
                 <width>131</width>
                 <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="volume_btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Установить!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Установите уровень громкости</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
       <rect>
           <x>10</x>
           <y>80</y>
           <width>221</width>
           <height>16</height>
       </rect>
   </property>
      <property name="text">
          <string>(Введите целое число от 1 до 100)</string>
      </property>
  </widget>
     <widget class="QPushButton" name="exit_btn">
         <property name="geometry">
             <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>75</width>
                 <height>23</height>
             </rect>
         </property>
         <property name="text">
             <string>Назад</string>
         </property>
     </widget>
     <widget class="QLabel" name="label_3">
         <property name="geometry">
             <rect>
                 <x>10</x>
                 <y>140</y>
                 <width>181</width>
                 <height>16</height>
             </rect>
         </property>
         <property name="font">
             <font>
                 <pointsize>11</pointsize>
             </font>
         </property>
         <property name="text">
             <string>Выберите цветовую тему</string>
         </property>
     </widget>
     <widget class="QWidget" name="">
         <property name="geometry">
             <rect>
                 <x>120</x>
                 <y>340</y>
                 <width>260</width>
                 <height>19</height>
             </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
                 <widget class="QRadioButton" name="radioButton_3">
                     <property name="text">
                         <string>Dark</string>
                     </property>
                 </widget>
             </item>
             <item>
                 <widget class="QRadioButton" name="radioButton_2">
                     <property name="text">
                         <string>Classic</string>
                     </property>
                 </widget>
             </item>
             <item>
                 <widget class="QRadioButton" name="radioButton">
                     <property name="text">
                         <string>White</string>
                     </property>
                 </widget>
             </item>
         </layout>
     </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
